demos/icon-toggle.html

```diff
@@ -178,7 +149,7 @@ <h2>Additional Color Combinations</h2>
                   favorite_border
                 </i>
               </div>
-              <div class="mdc-theme--text-primary-on-dark">Custom color on dark background</div>
+              <div class="mdc-theme--text-primary-on-primary">Custom color on background</div>
```

Comment:
I would suggest removing the background color (see my next comment), which means we can remove all CSS classes from this element. E.g.:

```html
<div>Custom color</div>
```